Services/User/classes/class.ilUserProfile.php

```diff
@@ -692,6 +692,10 @@ function addStandardFieldsToForm($a_form, $a_user = NULL, array $custom_fields =
 					if (ilUserProfile::userSettingVisible($f))
 					{
 						$em = new ilEMailInputGUI($lng->txt($lv), "usr_".$f);
+						// cat-tms-patch start 735
```

Comment:
Ist in emailinput ein default für die Länge gesetzt?

Reply:
Für max_length und size gibt es Konstanten in der GUI die im Standard gezogen werden.

Reply:
Ist es vielleicht schlauer die constante hoch zu setzen?

Reply:
Damit würde man dann ja alle EMailInputs in ganz ILIAS auf 140 erhöhen. Frage ist, ob das nötig ist.

Reply:
Ich werde es mergen, aber es wäre natürlich sinnvoll, da sobal jemand mit seiner Mail registriert ist, diese Person auch überall seine Mail eingeben können sollte...